x/ecocredit/client/basket/query.go

```diff
@@ -47,7 +47,7 @@ func QueryBasketsCmd() *cobra.Command {
 		Short: "Retrieves all baskets",
 		Long:  "Retrieves all baskets currently in state, with optional pagination",
 		Example: strings.TrimSpace(`regen q ecocredit baskets
-	regen q ecocredit baskets --pagination.offset 1 --pagination.limit 10`),
+	regen q ecocredit baskets --offset 1 --limit 10`),
```

Comment:
I'm not sure we want to use `strings.TrimSpace` when defining `Example`. Would be nice to remove here and to use the same format as the examples below.